build/teamcity/cockroach/nightlies/cloud_unit_tests_impl.sh

```diff
@@ -22,29 +22,37 @@ export AWS_SHARED_CREDENTIALS_FILE="$PWD/.aws/credentials"
 export AWS_CONFIG_FILE="$PWD/.aws/config"
 log_into_aws
 
+bazel_test_env = "--test_env=GO_TEST_WRAP_TESTV=1 \
```

Comment:
This should be `bazel_test_env="...` (with no spaces) otherwise it'll probably fail with something like `bazel_test_env: command not found`

Reply:
Thanks!